packages/paste-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,423 @@
+
+<DoDont>
+  <Do title="Do" body="Use a clear and concise heading to describe the purpose of the Modal." preview={false} />
+  <Dont title="Don't" body="Don't go into unnecessary detail in the heading about the purpose of the Modal." preview={false} />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Keep the main call to action in the footer. If you're asking the user to perform an action, use a primary action (primary button or destructive button). If you need only to give the user a way to close the Modal, use a secondary button or rely on the close 'x' button." preview={false} />
+  <Dont title="Don't" body="Don't put buttons that close the Modal, or confirm and submit content, in the body." preview={false} />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use at most one primary and one secondary action in a Modal. Make the secondary action a 'Cancel' button, especially when you want to give users an explicit choice to decline the primary action. This is especially useful when the primary action is a destructive action." preview={false} />
+  <Dont title="Don't" body="Don't put more than one primary action and one secondary action in the footer. Avoid using the secondary button for an action that isn't 'Close' or 'Cancel' since users will be used to pressing a secondary button that escapes the Modal. If you need to provide another action, add a third button to the footer, or consider showing it elsewhere." preview={false} />
+</DoDont>
+
+## Anatomy
+
+### General
+
+| Property         | Default token                     | Modifiable? |
+| ---------------- | --------------------------------- | ----------- |
+| background-color | $color-background-body            | No          |
+| border-color     | $color-background-body            | No          |
+| border-radius    | $border-radius-20                 | No          |
+| drop-shadow      | $shadow-card                      | No          |
+| modal-width      | sm [default]: 608px, lg: 816px    | No          |
+| overlay-color    | palette-gray-100 (at 50% opacity) | No          |
+
+### ModalHeader
+
+| Property                 | Default token       | Modifiable?  |
+| ------------------------ | --------------------| ------------ |
+| header-title-size        | $font-size-60       | No           |
+| header-title-weight      | $font-weight-medium | No           |
+| header-title-line-height | $line-height-40     | No           |
+| close-icon-color         | $color-gray-50      | No           |
+| close-icon-size          | $size-icon-24       | No           |
+| padding                  | $space-50           | No           |
+| bottom-border-width      | $border-width-10    | No           |
+| bottom-border-color      | $color-gray-50      | No           |
+
+### ModalBody
+
+| Property     | Default token       | Modifiable?  |
+| -------------| --------------------| ------------ |
+| inner-shadow | Need new token here | No           |
```

Comment:
This was in the spec doc. Not sure what token, if any, should be here.

Reply:
I didn't do it in the end. Showing a shadow only when content is scrollable seems a little too hard right now.